apps/web/src/pages/api/feed/userData.ts

```diff
@@ -0,0 +1,24 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+
+import { CACHE_TIMES } from 'src/constants/cacheTimes'
+import { getFarcasterProfile } from 'src/data/farcaster/queries/farcasterProfile.ts'
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
can we remove this api route now that we're fetching this data on the initial feed query?

Reply:
For sure!